documentation/service-rules/ecf1/transfers.md

```diff
@@ -0,0 +1,232 @@
+
+#### Pagination parameters (`Pagination`)
+
+| Name     | Type    | Required | Description                |
+| -------- | ------- | -------- | -------------------------- |
+| page     | integer | No       | Page number to return      |
+| per_page | integer | No       | Number of records per page |
+
+#### Response (200 OK)
+
+```json
+{
+  "data": [
+    {
+      "id": "db3a7848-7308-4879-942a-c4a70ced400a",
+      "type": "participant-transfer",
+      "attributes": {
+        "updated_at": "2021-05-31T02:22:32.000Z",
+        "transfers": {
+          "training_record_id": "000a97ff-d2a9-4779-a397-9bfd9063072e",
+          "transfer_type": "new_provider",
+          "status": "complete",
+          "leaving": {
+            "school_urn": "123456",
+            "provider": "Old Institute",
+            "date": "2021-05-31"
+          },
+          "joining": {
+            "school_urn": "654321",
+            "provider": "New Institute",
+            "date": "2021-06-01"
+          },
+          "created_at": "2021-05-31T02:22:32.000Z"
+        }
+      }
+    }
+  ]
+}
+```
+
+### GET /api/v3/participants/ecf/{id}/transfers
+
+This endpoint allows you to retrieve a single participant’s transfers
+
+#### Request parameters
+
+| Parameter | Type   | Required | Description                    |
+| --------- | ------ | -------- | ------------------------------ |
+| id        | string | Yes      | The ID of the ECF participant. |
+
+#### Response (200 OK)
+
+```json
+{
+  "data": {
+    "id": "db3a7848-7308-4879-942a-c4a70ced400a",
+    "type": "participant-transfer",
+    "attributes": {
+      "updated_at": "2021-05-31T02:22:32.000Z",
+      "transfers": [
+        {
+          "training_record_id": "000a97ff-d2a9-4779-a397-9bfd9063072e",
+          "transfer_type": "new_provider",
+          "status": "complete",
+          "leaving": null,
+          "joining": null,
+          "created_at": "2021-05-31T02:22:32.000Z"
+        }
+      ]
+    }
+  }
+}
+```
+
+## Transfer Fields
+
+The following describe how the transfer results are generated:
+
+- `id` - is set to `user.id`
+- `updated_at` - most recent `updated_at` from leaving or joining induction record, fallback to `user.updated_at`
+- `training_record_id` - is set to `participant_profile.id`
+- `transfer_type` - `induction_record.induction_programme.partnership.lead_provider` is used to calculate this value
+- `status` - if leaving end date or joining start date is in the past it is set to `incomplete`, otherwise `complete`
+- `created_at` - set to `leaving_induction_record.created_at`
+- `school_urn` - set to `induction_record.induction_programme.school_cohort.school.urn`
+- `provider` - set to `induction_record.induction_programme.partnership.lead_provider.name`
+- `date` - set to `end_date` for leaving and `start_date` for joining induction record.
+
+## Types of Transfers
```

Comment:
Is it worth documenting the logic around how transfers are compiled from induction records in the `BuildTransfers` service? it makes my head hurt every time I look at it so it might be nice to have it written down/explained somewhere for when we come to port it over to RECT 😅